docs/learn/documentation/versioned/jobs/samza-configurations.md

```diff
@@ -273,6 +273,8 @@ These properties define Samza's storage mechanism for efficient [stateful stream
 |stores.**_store-name_**.<br>rocksdb.max.log.file.size.bytes|67108864|The maximum size in bytes of the RocksDB LOG file before it is rotated.|
 |stores.**_store-name_**.<br>rocksdb.keep.log.file.num|2|The number of RocksDB LOG files (including rotated LOG.old.* files) to keep.|
 |stores.**_store-name_**.<br>rocksdb.metrics.list|(none)|A list of [RocksDB properties](https://github.com/facebook/rocksdb/blob/master/include/rocksdb/db.h#L409) to expose as metrics (gauges).|
+|stores.**_store-name_**.<br>side.inputs|(none)|For Samza applications with large stores that are periodically populated by a secondary data sources such as HDFS, but otherwise ready-only; Side inputs feature makes it easier and efficient to populate data. Stores configured with side inputs use the the source streams to bootstrap in case of failure thereby, reducing additional copy of the data in changelog. The value is a comma-separated list of streams.<br> Each stream is of the format `system-name.stream-name`. Additionally, applications should add the side inputs to job inputs (`task.inputs`) and configure side input processor (`stores.store-name.side.inputs.processor.factory`).
```

Comment:
@bharathkk 
In the implementation PR for this feature, we decided not to add side-input configurations to the public configuration reference(Thread here: https://github.com/apache/samza/pull/570#discussion_r204182323). Just wondering why are we adding it now?

Reply:
Because we've had time to test the implementation and are more confident about it. We'll add detailed feature documentation to the website later.